devicetypes/smartenit/iot8-z.src/iot8-z.groovy

```diff
@@ -0,0 +1,238 @@
+/**
+ *  Virtual IOT8Z
+ *
+ *  Copyright 2021 Luis Contreras
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ */
+
+metadata {
+	definition (name: "IOT8-Z", namespace: "Smartenit", author: "Luis Contreras", cstHandler: true, mnmn: "SmartThingsCommunity", vid: "6d510b74-469c-3fa0-be7a-ec0894d38dcb") {
+		capability "Contact Sensor"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Health Check"
+        capability "Sensor"
+        capability "Switch"
+        capability "monthpublic25501.analogSensor"
+        
+        command "childOn", ["string"]
+        command "childOff", ["string"]
+        
+        fingerprint manufacturer: "Smartenit, Inc", model: "IOT8-Z", deviceJoinName: "SMRT-IOT ", profileId: "0104", inClusters: "0000, 0003, 0006, 000C, 000F", outClusters: "0019" 
```

Comment:
I'm assuming this is switch device, so change `deviceJoinName` to `"Smartenit Switch"`